Networking/Networking/Model/ShippingLabel/Packages/WooShippingOriginAddress.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import Codegen
+
+public struct WooShippingOriginAddress: Equatable, GeneratedFakeable, GeneratedCopiable {
+    public let id: String
+    public let company: String?
+    public let address1: String
+    public let address2: String?
+    public let city: String
+    public let state: String?
+    public let postcode: String
+    public let country: String
+    public let phone: String
+    public let firstName: String
+    public let lastName: String
+    public let email: String
+    public let defaultAddress: Bool
+    public let isVerified: Bool
```

Comment:
I am not sure which properties should be mandatory and which ones should be optional?

Reply:
Let's check with the extension team before we finalize this. The existing `ShippingLabelAddress` model has all non-optional properties, which makes me think the backend might expect all of the properties (but e.g. with empty strings when they have no value). It'd be good to confirm that sooner rather than later.